node/tests/k8st/infra/values.yaml

```diff
@@ -1,40 +1,16 @@
-# The Calico version to use when generating manifests.
-version: latest-amd64
-
-# Configure the images to use when generating manifests.
-node:
-  image: docker.io/library/node
-calicoctl:
-  image: docker.io/calico/ctl
-typha:
-  image: docker.io/calico/typha
+installation:
   enabled: true
```

Comment:
this line pertained to `typha:`, the operator takes care of that automatically, right? I think this should be removed...